staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go

```diff
@@ -414,9 +414,9 @@ func NewDefaultAPIGroupInfo(group string, registry *registered.APIRegistrationMa
 		GroupMeta:                    *groupMeta,
 		VersionedResourcesStorageMap: map[string]map[string]rest.Storage{},
 		// TODO unhardcode this.  It was hardcoded before, but we need to re-evaluate
-		OptionsExternalVersion: &schema.GroupVersion{Version: "v1"},
-		Scheme:                 scheme,
-		ParameterCodec:         parameterCodec,
-		NegotiatedSerializer:   codecs,
+		//OptionsExternalVersion: &schema.GroupVersion{Version: "v1"},
```

Comment:
Hrm - we will need this in the future - it's intended to control what version of generic APIs are defaulted by the server, like the forthcoming Table.

Meta types should be registered in your group for now - is there a reason you don't want to?